transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java

```diff
@@ -236,18 +234,18 @@ public void testSelectTransferMethod_verifyTransferMethodsList() {
                 matches(atPosition(2, hasDescendant(wireAccountViewMatchers))));
 
         String wireTransferFee = InstrumentationRegistry.getInstrumentation().getTargetContext()
-                .getString(R.string.feeAndProcessingTimeInformation,usdDollarSymbol + "20.00","1-3 Business days");
+                .getString(R.string.feeAndProcessingTimeInformation, usdDollarSymbol + "20.00", "1-3 Business days");
 
         onView(withId(R.id.select_transfer_method_types_list)).check(
                 matches(atPosition(2, hasDescendant(withText(wireTransferFee)))));
 
         onView(withId(R.id.select_transfer_method_types_list)).check(
                 matches(atPosition(3, hasDescendant(withText(R.string.paypal_account_font_icon)))));
         onView(withId(R.id.select_transfer_method_types_list)).check(
-                matches(atPosition(3, hasDescendant(withText(R.string.paypal_account)))));
+                matches(atPosition(3, hasDescendant(withText("PayPal")))));
```

Comment:
why change to Hard coded string?

Reply:
previous its shown as Paypal Account , now from Api it's showing as  Paypal to fix the failed UI test I am hardcoded, Paypal string is not available in string.xml, added for this?

Reply:
api is to list the available transfer methods based on Country/Currency respond is indeed PayPal. I think why it was failed because before the string value for that key is indeed "PayPal". l but now we updated to PayPal Account. that's why it failed.

Reply:
Our mock data is this, and I checked with the actual TMC call , it shows PayPal as of today.
 {
                      "code": "PAYPAL_ACCOUNT",
                      "name": "PayPal",
                      "fees": {
                        "nodes": [
                          {
                            "country": "CA",
                            "currency": "CAD",
                            "transferMethodType": "PAYPAL_ACCOUNT",
                            "value": "0.25",
                            "feeRateType": "FLAT"
                          }
                        ]
                      }